packages/frontend/components/MessageCards/MessageCard.module.scss

```diff
@@ -0,0 +1,58 @@
+@import 'styles/variables';
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@700;800&display=swap');
+
+.container {
+  background: $background;
+  overflow: hidden;
+  justify-content: center;
+  margin: 80px;
+  padding: 20px;
+  align-content: flex-start;
+}
+
+.card {
+  background-color: $orange;
+  aspect-ratio: auto;
+  border-radius: 10px;
+  color: $white;
+  font-family: 'Open Sans';
+  padding: 5px;
+  position: relative;
+  overflow: visible;
+
+  .title {
+    text-align: left;
+    margin-top: 12px;
+    font-size: 20px;
+    margin-left: 49px;
+    font-family: 'Open Sans';
+  }
+
+  img {
+    float: right;
+    position: relative;
+    top: -60px;
+    left: 10px;
+    height: 105px;
+  }
+
+  .message {
+    font-size: 24;
```

Comment:
Missing unit. I think you mean `px`, but that seems to be a bit too big compared to what you had before. `16px` works fine.